Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,87 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure prepares the telescope and dome to take flatfileds and describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
```

Comment:
Typo *flatfields*
Please consider something like this for the overview:
"This procedure prepares the telescope and dome to take flat fields, pointing the telescope to the dome flat screen. " 
I would remove "describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters." as this procedure doesn't execute flat field calibrations.